src/Game.cpp

```diff
@@ -51,6 +51,8 @@
 #include "GameWindow.h"
 #include "Graphics/FrameBuffer.h"
 #include "Graphics/Texture2D.h"
+#include "Input/GameActionMap.h"
```

Comment:
Not a big fan of including this. Maybe a factory function in GameActionMapInterface would be better.